### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by Johnmon40**
## Description

Please include a summary of the changes and which issue is fixed (if applicable).

Fixes # (issue)
Closes # (issue)

## Type of Change

Please select the relevant option(s):

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation update
- [ ] 🔨 Refactoring (no functional changes)
- [ ] ⚡ Performance improvement
- [ ] ✅ Test update
- [ ] 🔧 Configuration change
- [ ] 🎨 Style/UI update

## Motivation and Context

Why is this change required? What problem does it solve?

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so reviewers can reproduce.

- [ ] Unit tests
- [ ] Integration tests
- [ ] Manual testing
- [ ] End-to-end tests

**Test Configuration**:
* OS: 
* Browser (if applicable): 
* Version: 

## Screenshots (if applicable)

Add screenshots to help explain your changes.

### Code Quality
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have removed any console.log or debugging code

### Testing
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have tested edge cases and error scenarios

### Documentation
- [ ] I have made corresponding changes to the documentation
- [ ] I have updated the README (if needed)
- [ ] I have added/updated code comments where necessary

### Dependencies
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked for and resolved any merge conflicts
- [ ] I have updated dependencies to their latest stable versions (if applicable)

### Security
- [ ] I have checked for security vulnerabilities
- [ ] I have not committed any sensitive information (passwords, API keys, etc.)

## Breaking Changes

Does this PR introduce any breaking changes? If yes, please describe the impact and migration path:

## Performance Impact

Does this change have any performance implications? (positive or negative)

## Additional Notes

Add any additional notes or context about the pull request here.

## Related Issues/PRs

Link any related issues or pull requests:
- Related to #
- Depends on #
- Blocks #

---

### Review Focus Areas

Please pay special attention to:
- 
- 

### Testing Instructions

How to test these changes:
1. 
2. 
3.

## Summary by Sourcery

Overhaul index.html to provide a self-contained static interface for MCA Lead Finder, replacing the React-based entrypoint with inline HTML, CSS, and vanilla JavaScript that handles search, status updates, and results display.

New Features:
- Add static HTML interface for MCA Lead Finder with header, footer, search controls, status indicator, and results table
- Implement client-side JavaScript to fetch search results from /api/search, render them in a table, handle clear actions and escape HTML safely

Enhancements:
- Remove React root and TSX entrypoint and replace external CSS with inline styles in index.html
- Update document metadata and title for the MCA Lead Finder tool

@copilot
@Johnmon40 original prompt "i want to make this a working website now, change the index to work and make it very user friendly. I want it to scrap the entire internet to generate fresh merchant cash advance leads that are currently looking for funding. Make it fully free to use and no sign up and have it search all data basis for the uccs and funding requests"

### CODE REVIEW

**Comment by 4444JPP** [comment]
@Johnmon40 original prompt "i want to make this a working website now, change the index to work and make it very user friendly. I want it to scrap the entire internet to generate fresh merchant cash advance leads that are currently looking for funding. Make it fully free to use and no sign up and have it search all data basis for the uccs and funding requests"

File: index.html [hunk 1]
```diff
@@ -1,19 +1,112 @@
...
+
+startBtn.onclick = async () => {
+  const q = document.getElementById('q').value.trim();
+  const source = document.getElementById('source').value;
+  if(!q){ alert('Type a search phrase first'); return; }
+  status.innerText = 'Searching...';
+  resultsBody.innerHTML = '';
+  resultsTbl.style.display = 'none';
+  try {
+    const res = await fetch(`/api/search?q=${encodeURIComponent(q)}&source=${encodeURIComponent(source)}`);
```
[the remark this answers is not in the document]
Comment (4444JPP) [reply index.html]: implement suggestion;

File: index.html [hunk 2]
```diff
@@ -1,19 +1,112 @@
...
-</body>
+  <header>
+    <div class="wrap">
+      <h1>MCA Lead Finder — Free · No Signup</h1>
+      <div class="small">Search public signals for Merchant Cash Advance & funding requests. Use responsibly.</div>
+    </div>
+  </header>
+
+  <main class="wrap">
+    <div style="margin-top:18px">
```
[the remark this answers is not in the document]
Comment (4444JPP) [reply index.html]: implement suggestion;

File: index.html [hunk 3]
```diff
@@ -1,19 +1,112 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>UCC-MCA Intelligence Platform</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Oswald&family=Roboto+Slab&family=Space+Mono&display=swap">
-    <link href="/src/main.css" rel="stylesheet" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <title>MCA Lead Finder — Free, No Signup</title>
+  <style>
```
[the remark this answers is not in the document]
Comment (4444JPP) [reply index.html]: implement suggestion;

File: index.html [hunk 4]
```diff
@@ -1,19 +1,112 @@
...
+<script>
+const startBtn = document.getElementById('start');
+const clearBtn = document.getElementById('clear');
+const status = document.getElementById('status');
+const resultsTbl = document.getElementById('results');
+const resultsBody = document.getElementById('resultsBody');
+
+startBtn.onclick = async () => {
+  const q = document.getElementById('q').value.trim();
+  const source = document.getElementById('source').value;
```
[the remark this answers is not in the document]
Comment (4444JPP) [reply index.html]: implement suggestion;

**Review by 4444JPP — APPROVED** [review]
